### ISSUE TITLE
100% translated document is showing as only 50% translated on the overview tab

### ISSUE THREAD

**Issue by steve4744** [gitlocalize/feedback#96]
!!PLEASE READ IT!!

To make it easier for us to treat your feedback, 
we have made a template you can fill out.

Before opening a new issue, make sure to search for keywords in the issues
filtered by the "bug" label.

- https://github.com/gitlocalize/feedback/labels/bug

and verify the issue you're about to submit isn't a duplicate. 
Please make separate posts for each feature request or bug report.
Please remove this notice if you're confident your issue isn't a duplicate.

If you require more general support please send emails:
info[@]gitlocalize.com

------

### Summary

Document is 100% translated and PR has been merged, but on the overview page is showing as 50%.

### Steps to reproduce

There is an Italian translation which is in progress, and yet to be reviewed. This says 23% translated, but on the overview page is 11%.

### Repository URL
https://gitlocalize.com/repo/5420

### What is the current *bug* behavior?

The overview tab is not showing the correct percentage of the fully translated document.
Looking at both the Spanish and Italian translations, the overview tab seems to be displaying half of what I would expect it to display.

### What is the expected *correct* behavior?

The overview tab should show 100% translated for a fully translated document.

### Relevant logs and/or screenshots

![es1](https://user-images.githubusercontent.com/6975392/103049664-55aaea80-458a-11eb-9dc2-7c10e1d0d72f.PNG)

![es2](https://user-images.githubusercontent.com/6975392/103049243-2182fa00-4589-11eb-97fb-13efebd2b97b.PNG)

### Possible fixes

(If you have any idea to solve the problem, please help us.)

**Comment by ilyaspiridonov** [comment]
Thanks for reporting this. Looks like some caching issue on our end. It appears we were able to fix this at the subfolder level (now 100%), but at the Language level it's still displaying %% incorrectly (in project overview). 
Will probably take some time, but we'll keep you posted :)

**Comment by steve4744** [comment]
I fixed my issue with this one particular translation. In case it helps, these are the steps I followed:

- I decided to delete the translation and redo it from the beginning, so
- deleted the Spanish translation from the "Manage Languages" page
- checked it had gone on the overview page
- back to "Manage Languages", chose "Spanish [es]" from the drop-down language list, and clicked "Add"
- selected the "Spanish" link to edit the file

When I opened the file it was already 100% translated(?). Went to the overview page and that was also now showing (correctly) 100% translated.
